### PR TITLE
Escaping and fix base64 on linux

### PR DESCRIPTION
**Pull request by schulzh**

### CODE REVIEW

**Comment by janmedrek** [comment]
Hey, thanks for the contribution, will review as soon as GitHub is back to being operational 👍

File: one-click-integration.sh [hunk 1]
```diff
@@ -176,7 +176,7 @@ echo ""
 ## Decode response and save as a certificate file
 echo "Decoding and saving certificate"
 echo ""
-crt=$( echo ${crtb64} | base64 --decode > generated.crt )
+crt=$(echo "${crtb64}" | base64 -d > generated.crt )
```
Comment (janmedrek) [inline one-click-integration.sh]: I've got an error running this on Mac OS, the -d option does not work. After the following change I was able to go through the whole flow:
```suggestion
crt=$(echo "${crtb64}" | base64 -D > generated.crt )
```

Is it also a viable solution for Linux?

Comment (schulzh) [reply one-click-integration.sh]: It does not, however we could use openssl for that: `openssl enc -base64 -d`

File: one-click-integration.sh [hunk 2]
```diff
@@ -176,7 +176,7 @@ echo ""
 ## Decode response and save as a certificate file
 echo "Decoding and saving certificate"
 echo ""
-crt=$( echo ${crtb64} | base64 --decode > generated.crt )
+crt=$(echo "${crtb64}" | openssl enc -base64 -d > generated.crt )
```
Comment (janmedrek) [inline one-click-integration.sh]: On my machine this only returns last line of the certificate (`-----END CERTIFICATE-----`), does this solution works on linux as well? Adding one-line processing made it for me.
```suggestion
crt=$(echo "${crtb64}" | openssl enc -base64 -d -A > generated.crt )
```

Comment (schulzh) [reply one-click-integration.sh]: Seems to work fine with Linux and Alpine openssl

**Pull request merged**